helm/aqua-app-scanner/values.yaml

```
@@ -25,6 +25,13 @@ logLevel: "INFO"
 
 user: ""
 password: ""
+
+userCredentials:
+  enable: false   # enable to true for using secret to load username and password for scanner
+  secretName: ""  #credentials secret name
+  userKey: ""     #username key defined in secret
+  passwordKey: "" #password key defined in secret
```

```suggestion
  passwordKey: ""  #password key defined in secret
```
Double white space due to linting rules